ipatests/test_integration/test_user_permissions.py

```diff
@@ -190,6 +190,36 @@ def test_user_add_withradius(self):
              '--user-auth-type', 'radius', '--radius-username', 'testradius',
              'testradius', '--radius', radiusproxy])
 
+    def test_user_add_fix_capitalization_error(self):
```

Comment:
- this would be better located in test_commands.py::TestIPACommand
- please change the name to test_user_mod_change_capitalization_issue5879()
- please add entries for this test in the nightlies

Reply:
@fcami , have made the required changes.